### PR TITLE
updating workshop

### PR DESCRIPTION
**Pull request by j2ktown**

### CODE REVIEW

File: cli/pom.xml
Comment (mik-rzo) [inline cli/pom.xml:1]: envious of your dependencies😍

File: cli/src/main/java/org/example/Main.java
Comment (mik-rzo) [inline cli/src/main/java/org/example/Main.java:1]: delete O'connell from the database 😉

**Review by mik-rzo — COMMENTED** [review]
nice work on the workshop

File: cli/pom.xml
Comment (j2ktown) [reply cli/pom.xml:1]: Thanks

File: cli/src/main/java/org/example/Main.java
Comment (j2ktown) [reply cli/src/main/java/org/example/Main.java:1]: ok

**Review by mik-rzo — APPROVED** [review]

**Review by Charlie-robin — APPROVED** [review]
Good to merge :)

**Pull request merged**